truffle-config.js

```diff
@@ -10,13 +10,15 @@ const networks = Object.assign(...[
   [4, 'rinkeby'],
   [5, 'goerli', `${2e9}`],
   [42, 'kovan'],
-].map(([networkId, network, gasPrice]) => ({
+  [77, 'sokol',, true],
```

Comment:
Note: we do not have sokol in the networks file

Reply:
Do we want to create a "Sokol deployment PR" or remove this?